source/use-cases/sensitive-data-encryption.txt

```diff
@@ -603,7 +643,24 @@ MedcoMD engineers write a function to create a new patient record:
                  console.log("writeError occurred:", writeError);
              })
          }
-
+   .. tab::
+      :tabid: python
+
+      .. code-block:: python
+
+         def insert_patient(collection, name, ssn, blood_type, medical_records, policy_number, provider):
```

Comment:
Nitpick: no action needed here, just a coding tip. Long parameter lists are considered a "code smell" that indicates it should be refactored. One simple way to reduce the number of parameters is to store them in a struct/dict/map and pass that as a parameter (similar to what you did with unpacking `extra_options`).

Reply:
Cool thanks for the tip, will keep that in mind for the future